Choose your course guidance (#3606)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

---

You need a bachelor's degree (in any subject) to teach in primary, secondary and special schools in England.

For secondary teacher training, your provider will want to make sure you have a good knowledge of the subject you’re applying to teach.

If you have a degree, you can do postgraduate teacher training to get qualified teacher status (QTS).

You need [English qualified teacher status (QTS)](/what-is-qts) to teach in most schools in England.

$international-content$

## Teacher training course providers

Your postgraduate teacher training course might be provided by: 

* a university (sometimes referred to as university-led training) 
* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)

> Some providers are ‘accredited’ – this means they've been given permission by the Department for Education (DfE) to run teacher training courses.

Reviewer: ```suggestion
Some providers are ‘accredited’ – this means they've been approved by the Department for Education (DfE) to run teacher training courses.
```

---

You need a bachelor's degree (in any subject) to teach in primary, secondary and special schools in England.

For secondary teacher training, your provider will want to make sure you have a good knowledge of the subject you’re applying to teach.

If you have a degree, you can do postgraduate teacher training to get qualified teacher status (QTS).

You need [English qualified teacher status (QTS)](/what-is-qts) to teach in most schools in England.

$international-content$

## Teacher training course providers

Your postgraduate teacher training course might be provided by: 

* a university (sometimes referred to as university-led training) 

> * a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 

Author: I think this is accurate, in that apprenticeships are technically delivered by schools? one to check

* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)

> Some providers are ‘accredited’ – this means they've been given permission by the Department for Education (DfE) to run teacher training courses.

Author: Included this section (reworded with the help of Kate) because candidates will encounter this kind of information when they go into Find, so I thought it was important to address on GIT to help them understand what any of this means!!

Reviewer: I think it is a good idea to include - is `permission` the right word though?

Might be worth checking with the accreditation team

Author: @Kateharries @MaeveRoseveare is 'permission' right here?

Reviewer 2: maybe 'approved'?

> Providers who have not been accredited can also run courses, but they'll be in a partnership with an organisation that has. For example, you might see a school running a course that has a university as their ‘accredited provider’.

Reviewer: ```suggestion
Providers who have not been accredited can also run courses if they are in a partnership with an organisation that has been accredited. For example, you might see a school running a course that has a university as their ‘accredited provider’.
```
I'm not sure that `but` makes sense here? However, my suggested wording might not be right!

> 

Reviewer 3: Would it make sense to add a link to 'How to choose your teacher training course' here?

Author: I'm not sure about here - I think given that this section is specifically about the course provider, but 'How to choose your teacher training course' is a bit wider than that, as in not just about the provider but qualifications, salary, etc. maybe not? but maybe elsewhere on the page?

## What does teacher training involve?

Postgraduate teacher training usually takes 9 months full-time, or 18 to 24 months part-time. 

All teacher training involves mostly classroom placements with some theoretical learning.  

This means that even if your course provider is a university, you will still do placements in schools. And if your course provider is a school, you might still do your theoretical learning in a university. 

The important thing to remember is that regardless of your course provider, all teacher training involves around 120 days of school placements in at least 2 schools.

You can [find out more about what to expect on your teacher training](/train-to-be-a-teacher/initial-teacher-training). 

## Where will my training take place?

Because all teacher training must involve placements in at least 2 schools, your training will not all happen in the same place. Some courses may also have a separate location for your theoretical learning. 

> Your course provider will probably not be able to confirm your exact placement locations until they’ve allocated all training places on their course. But you can talk to them beforehand to get a rough idea.

Reviewer: ```suggestion
Your course provider will probably not be able to confirm your exact placement locations until they’ve allocated all training places on their course. But you can talk to them beforehand to get an idea.
```

## How much does teacher training cost?

Postgraduate teacher training course fees are around £9,250 per year (or higher for non-UK citizens). 

You can apply for tuition fee and maintenance loans, even if you already have a student loan. 

You may be eligible for extra financial support if:

> * you train to teach certain subjects

Author: Decided to do one big link to the funding section rather than lots of little links to the specific content pages - mainly did this to reduce the link load, but actually not sure which approach is best?

* you're disabled
* you're a parent or carer
* you're a veteran
* you're eligible for a salaried teacher training course

> [Find out more about how to fund your training](/funding-and-support).

Reviewer: ```suggestion
[Find out how to fund your training](/funding-and-support).
```
